docroot/profiles/custom/cgov_site/modules/custom/cgov_blog/src/Services/BlogManager.php

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
If you are not using the manager can you please remove it?

Reply:
Manager class removed.